docs/source/guides/getting_started.rst

```diff
@@ -334,33 +334,54 @@ AwesomeThingModule
 
   public class AwesomeThingModule extends PrivateModule {
     protected void configure() {
-      bind(HelloWorldResource.class).to(HelloWorldResourceImpl.class);
+      bind(HelloWorldResource.class);
+
+      // Note: All clients that will be used should be installed in this module to avoid registering
+      // client JAX-RS resources with the server. The Correlation ID header name for clients could
+      // also be configured/overridden here.
 
       expose(HelloWorldResource.class);
     }
   }
 
-``AwesomeThingModule`` is a private module, which contains configuration information for our service. PrivateModule encapsulates the service's environment and only exposes objects needed for JAX-RS/Resteasy(ex: resource classes, providers and features). It also prevents dependency issues with a configuration bound to Guice gobally.
+``AwesomeThingModule`` is a private module, which contains configuration information for our service.
+PrivateModule encapsulates the service's environment and only exposes objects needed for
```

Comment:
'PrivateModule encapsulates' -> 'The PrivateModule encapsulates'

Reply:
https://github.com/cerner/beadledom/pull/133/commits/90d88965913cd8d3fcb84c756c7d2516e8aa2506